tg_odesli_bot/bot.py

```diff
@@ -375,9 +375,12 @@ def _compose_reply(
         """
         # Quote the original message for group chats
         if message.chat.type != ChatType.PRIVATE:
-            reply_list = [
-                f'<b>@{message.from_user.username} wrote:</b> {message_text}'
-            ]
+            if message.from_user.username:
+                reply_list = [f"<b>@{message.from_user.username} wrote:</b> {message_text}"]
+            else:
+                reply_list = [
+                    f"<b><a href='tg://user?id={message.from_user.id}'>{message.from_user.full_name}</a>  wrote:</b> {message_text}"
```

Comment:
There is a double whitespace before `wrote:</b>`. The line seems to be too long as well.

Reply:
Were fixed in [6b725f](https://github.com/9dogs/tg-odesli-bot/pull/21/commits/6b725f8023926ba0068a9ae7b3d0cfad602f467f)

Reply:
I've stumbled upon the exact functionality we're trying to replicate: [`get_mention`](https://github.com/aiogram/aiogram/blob/7ad756658da8d0646562287a00858a6e2af81679/aiogram/types/user.py#L70). We need something like that:
```python
if message.chat.type != ChatType.PRIVATE:
    user_mention = message.from_user.get_mention(as_html=True)
    reply_list = [
        f'<b>{user_mention} wrote:</b> {message_text}'
    ]
else:
```

Reply:
Yes, this indeed works, I just checked.
[PR](https://github.com/9dogs/tg-odesli-bot/pull/21/commits/3959edcc807bf80b33bfcd0b6418bc57df871213) for fix is submitted